components/script/dom/canvas2d/mod.rs

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+pub mod canvas_state;
```

Comment:
Canvas state isn't a DOM interface and shouldn't be here.

Reply:
Hm, we need it just in this module (i think this is `pub(super)`)